.circleci/config.yml

```diff
@@ -751,15 +753,12 @@ workflows:
     - gradle-check-jdk11:
         requires:
         - hold
-        - pr-approval/authenticate
```

Comment:
Does this mean PRs will run without authenticate from your side?

Reply:
No; this is a job that auto-approves the `hold` job if the commit was authored by a maintainer.

This job has been a little flakey for us, though, and it's been preventing us from testing pull requests due to being a dependency for the actual test.

This gets rid of that dependency (it doesn't actually need to be a dependency)